packages/celltags-extension/package.json

```diff
@@ -40,10 +40,10 @@
   "dependencies": {
     "@jupyterlab/application": "^2.0.0-alpha.4",
     "@jupyterlab/celltags": "^2.0.0-alpha.4",
-    "@jupyterlab/notebook": "^2.0.0-alpha.4",
-    "@types/node": "^12.12.17"
+    "@jupyterlab/notebook": "^2.0.0-alpha.4"
   },
   "devDependencies": {
+    "@types/node": "^12.12.17",
```

Comment:
@blink1073 I was thinking through how to deal with this. Thanks for handling it!

I think there's an argument for [getting rid of any explicit `@types/node` declaration](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types), but I'll save that for another PR